server/static/index.html

```diff
@@ -0,0 +1,7 @@
+<!doctype html>
```

Comment:
Let's move this data into a string inside the server JS for now: it saves us from using `serve-static` module altogether, and will allow better control later.